frappe/desk/page/setup_wizard/install_fixtures.py

```diff
@@ -28,4 +31,135 @@ def update_genders_and_salutations():
+                            lft = lft + 1
+                    lft = lft + 1
+            lft = lft + 1
+
+
+def make_administrative_areas():
+    """
+    ASSUMPTIONS: 1. Administrative_areas.json file does not have complete duplicates
+                 2. There are only 5 levels i.e.country, state, county, city and pincode (hardcoded names)
+                 3. In administrative_areas.json child occures after parent
+
+    """
+    #enter country administrative area as root node with no parent
+    country = frappe.db.get_value("System Settings", "System Settings", "Country")
+    if country == "India":
+        with open("../apps/frappe/frappe/geo/administrative_regions/india.json") as f:
+            administrative_areas = json.loads(f.read())
+        country_record = {
+            "doctype": "Administrative Area",
+            "title": country,
+            "administrative_area_type": "country",
+            "is_group": 1,
+            "lft": 0,
+            "rgt": 2*len(administrative_areas) + 1
+        }
+        make_fixture_record(country_record)
+        for record in administrative_areas:
+            record.update({
+                "parent_unique_name": "".join(record['parent']),
+                "self_unique_name": "".join(record['parent']) + "" + record['title']
+            })
+
+        # Calculate lft, rgt and later use db_insert() to speed up
+        bulid_administrative_area_tree(administrative_areas, country)
+
+        # add other administrative areas for the country selected in setup if data available in aa.jaon file
+
+        administrative_areas_copy = copy.deepcopy(administrative_areas)  # copy by value
+        for record in administrative_areas:
+            record.update({
+                "doctype": "Administrative Area",
+                "parent_administrative_area": get_parent_name(country, record, administrative_areas, administrative_areas_copy),
+                "is_group": 1
+            })
+            del record['parent']
+            del record['parent_unique_name']
+            del record['self_unique_name']
+            make_fixture_record(record)
+
+
+def get_parent_name(country, record, administrative_areas, administrative_areas_copy):
+    if record['parent_unique_name'] == "":
+        return country.title()
+    else:
+        parent_details = [area for area in administrative_areas_copy if area['self_unique_name'] == record['parent_unique_name']]
+        if len(parent_details) == 0:
+            frappe.throw("wrong parent hierarchy")
+        elif len(parent_details) > 1:
+            frappe.throw("duplicate entry")
+        else:
+            parent_name = [admin_area for admin_area in administrative_areas if parent_details[0]['lft'] == admin_area['lft'] and parent_details[0]['rgt'] == admin_area['rgt'] and parent_details[0]['title'] == admin_area['title']]
+            return parent_name[0]['name']
+
+
+def make_fixture_record(record):
+    from frappe.modules import scrub
```

Comment:
tabs, not spaces!

Reply:
Replaced spaces by tabs in install_fixtures.py